mobile/src/bundles/auth/components/auth-wrapper/auth-wrapper.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { Image, StatusBar, View } from '~/bundles/common/components/components';
+import { Color } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type Properties = {
+    children: React.ReactNode;
+};
+
+// eslint-disable-next-line unicorn/prefer-module, @typescript-eslint/no-var-requires
+const logoImg = require('~/assets/images/logo.png');
```

Comment:
I am not sure about `require`, we should use modules

Reply:
Am I allowed create index.d.ts file in the mobile src folder?